src/core_modules/capture-core/components/PossibleDuplicatesDialog/possibleDuplicatesDialog.actions.js

```diff
@@ -0,0 +1,50 @@
+// @flow
```

Comment:
I am extracting here the actions related to the duplicates dialog. We then use these action creators in the hook `useDuplicates` below.

Having the actions creators returned from the `useDuplicates` hook allow us to reuse them in the different places that are needed.